modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -170,3 +174,13 @@ the correct props.
 We're removing memoization from focus ring. The `memoize` argument passed to `focusRing` is no
 longer valid and we've removed the exported `memoizedFocusRing`. There is no codemod for this
 change. We couldn't find any example of `memoize` or `memoizedFocusRing` being used.
+
+## Token Updates
+
+### depth
+
+In `v7` we released an update to our `depth` tokens that were too bold and harsh for web applications. We've modified the `depth` token values
+to be more subtle and improve visual design. These updates change the `box-shadow` value and not the `depth` value used pero component. These changes
+affects all components that use a box shadow including: `Card, Toast, Dialog, Popup, Modal, Menu`.
+
+<img src={depthTokenUpdate} atl="A before and after of depth token values" />
```

Comment:
I don't think this will work on the Canvas Site since [build-mdx](https://github.com/Workday/canvas-kit/blob/prerelease/major/modules/docs/utils/build-mdx.js) doesn't include images in the canvas-kit-docs package (it'll show up as a broken image on the site).

I agree that this image is useful, though. I know we want to get this PR merged so we can test the depth changes with other teams next sprint -- I'd remove the image for now and save it for a follow-up PR after we've figured out how to update build-mdx and the site to support images referenced by canvas-kit-docs MDXes. In the meantime, we could include this image in the v9 announcement on the site.

Reply:
Fair point! Thanks James!